### PR TITLE
Book store

### PR DESCRIPTION
**Pull request by tasnimchy96**
In this branch we:

- Build a basic website that allows users to add/remove books from a list.  
- Used JavaScript objects and arrays.
- Dynamically modified the DOM and add basic events.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,117 @@
...
+  - [Project-Presentation](#project-presentation)
+  - [Live-Demo](#live-demo)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# :book: [Portfolio]
```
Comment (TedLivist) [inline README.md]: - [x] Kindly fix your project title.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @tasnimchy96 and @samreenmj,

Good job so far!
✔️ Books can be added and removed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (tasnimchy96) [reply README.md]: Done

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @tasnimchy96   👋 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- Working code ✅ 
- Moderate interface ✅ 


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**